toolchains/solidity/core/crates/linter-cli/src/main.rs

```diff
@@ -100,6 +109,7 @@ fn main() -> Result<(), SolidHunterError> {
     if args.path.is_empty() {
         let mut linter: SolidLinter = SolidLinter::new();
         linter.initialize_rules(&args.rules_file)?;
+        linter.initialize_ignore_file(&args.ignore_file)?;
```

Comment:
With the new usage (```./solidhunter ./path/to/file-or-project```) , the .solidhunterignore can be found at the root of the project path, it is not always `.solidhunterignore` from where the binary is launched.